operator/builtin/input/syslog/syslog_test.go

```diff
@@ -26,6 +25,8 @@ import (
 	"github.com/open-telemetry/opentelemetry-log-collection/pipeline"
 	"github.com/open-telemetry/opentelemetry-log-collection/testutil"
 	"github.com/stretchr/testify/require"
+
+	"fmt"
```

Comment:
Why does this need to move? Seems it belongs with other native packages.

Reply:
haha, don't know why too. just goimports auto fix it. let me research it

Reply:
look like goimports is stupid when having empty line.
try to use gci(include in golangci-lint too) instead of goimports
https://github.com/daixiang0/gci